src/main/java/com/product/service/ProductService.java

```diff
@@ -8,11 +8,11 @@
 
 public interface ProductService {
 
-	public void deleteProductById(int id);
+	public void deleteProductbyId(int id);
 
 	public void updateProduct(Product emp);
 
-	public void deleteAllProducts();
+	public void deleteallProducts();
 
```

Comment:
It should be the previous one.. deleteAll....

Reply:
Done.